index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Development course</title>
+</head>
+
+<body>
+  <div class="mobile-logo-wrap">
+    <div>
+      <a href="index.html">
+        <img src="Images/main-logo.png" alt="developer week global">
+      </a>
+    </div>
+  </div>
+
+  <div class="mobile-menu-wrap">
+    <div class="mobile-menu-button">
+      <a href="#" id="mobile-menu-button">
+        <img src="Images/mobile_menu.png" alt="Mobile menu button">
+      </a>
+    </div>
+
+    <div class="mobile-menu-display display-none" id="mobile-menu-display">
+      <div class="mobile-menu-item align-self-fx-end">
+        <button id="close-mobile-menu">&#10799;</button>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>News</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Sponsor</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Join</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Program</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="about.html">
+          <p>About</p>
+        </a>
+      </div>
+    </div>
+  </div>
+  <!-- Desktop navbar  -->
```

Comment:
- [ ] This section of the navbar is currently missing in your UI 👇🏽 .

![image](https://user-images.githubusercontent.com/63473584/197045841-53b953ac-da21-4570-a9b8-e95fc3e976d2.png)

Kindly add it so that your design will match the design template given because, in a professional working environment, you will be expected to stick to design as much as possible. So practice it now 😄